jsonschema-core.xml

```diff
@@ -1767,6 +1767,10 @@
                             URI to identify more than one schema. When multiple schemas try to identify
                             as the same URI, validators SHOULD raise an error condition.
                         </t>
+                        <t>
+                            As with all schemas, a referenced schema SHOULD be processed according to its define "$schema"
```

Comment:
Not to diminish @handrews' excellent comments, but maybe remove the initial "define" as it reads somewhat redundant with the later "if defined."

Reply:
Yeah makes sense